bigquery_etl/shredder/bug1751979.py

```diff
@@ -249,3 +249,119 @@
     )
 ) AS payload)
 """
+
+BUG_1751979_MAIN_SUMMARY_V4_UDFS = r"""
+CREATE TEMP FUNCTION sanitize_search_counts_ms(
+  input ARRAY<STRUCT<engine STRING, source STRING, count INT64>>
+) AS (
+  (
+    WITH base AS (
+      SELECT
+        -- This reverses the separation logic in main_summary where
+        -- engine and source are parsed from the search_counts key
+        CONCAT(engine, '.', source) AS key,
+        count
+      FROM
+        UNNEST(input)
+    ),
+    parsed AS (
+      SELECT
+        *,
+        REGEXP_EXTRACT(key, "([^.]+\\.in-content[:.][^:]+:).*") AS prefix,
+        REGEXP_EXTRACT(key, "[^.]+\\.in-content[:.][^:]+:(.*)") AS code,
```

Comment:
I think it would better not to rely on being escaped the right number of times:
```suggestion
        REGEXP_EXTRACT(key, "([^.]+[.]in-content[:.][^:]+:).*") AS prefix,
        REGEXP_EXTRACT(key, "[^.]+[.]in-content[:.][^:]+:(.*)") AS code,
```